components/list/list-item-mixin.js

```diff
@@ -387,8 +409,8 @@ export const ListItemMixin = superclass => class extends composeMixins(
 
 			[slot="add"],
 			[slot="add-top"] {
-				margin-bottom: -4px;
-				margin-top: -3px;
+				margin-bottom: -12.5px;
```

Comment:
How does the browser handle the half pixels? Is this just the same as `-12px` or `-13px`?

Reply:
On a separate note, hopefully these negative margins don't cause us any issues. I think we had an issue in the past where a `-1px` margin was causing scrollbars to appear somewhere, possibly in a scrollable container.

Reply:
I'll re-test these numbers. Eek I also hope not! Will keep an eye on that.